packages/ui/src/plate-ui/mention-input-element.tsx

```diff
@@ -100,7 +100,7 @@ export const MENTIONABLES = [
     key: '12',
     text: 'Aiolin and Morit Astarte',
   },
-  { key: '13', text: 'Aks Moe' },
+  { key: '13', text: 'Ask Moe' },
```

Comment:
you sure this is a correct spell change?

Reply:
it's fake text that doesn't actually matter so I decided to just let it spell it however it wanted